tests/compilers/tests/version_match

```diff
@@ -35,29 +35,27 @@ family=$LMOD_FAMILY_COMPILER
     local my_mod_version=`cat .cmd_output | sed '/^\s*$/d' | sed 's/^[ \t]*//' | tail -1 | awk -F "$my_family/" '{print $2}'`
 
     if [[ "$my_family" = "intel" ]];then
-	# intel has no consistency here, have to punt
-## 	compiler=`which icc`
-## 	version=`icc --version | head -1 | awk '{print $3}'`
-## 	assert_equal "$my_mod_version" "$version"
-	version="assuming all is good"
-  elif [[ "$my_family" =~ "arm1" ]];then
-	assert_equal "$my_mod_version" "compat"
+      compiler=`which icx`
+      version=`icx --version | awk 'sub(/^.* 202/,"202",$0){print $1;exit}'`
+      assert_equal "$my_mod_version" "$version"
+    elif [[ "$my_family" =~ "arm1" ]];then
+	  assert_equal "$my_mod_version" "compat"
     elif [[ "$my_family" =~ "gnu"  ]];then
-	version=`gcc --version | head -1 | awk '{print $3}'`
-	assert_equal "$my_mod_version" "$version"
-	version=`g++ --version | head -1 | awk '{print $3}'`
-	assert_equal "$my_mod_version" "$version"
-	version=`gfortran --version | head -1 | awk '{print $4}'`
-	assert_equal "$my_mod_version" "$version"
+	  version=`gcc --version | head -1 | awk '{print $3}'`
+	  assert_equal "$my_mod_version" "$version"
+	  version=`g++ --version | head -1 | awk '{print $3}'`
+	  assert_equal "$my_mod_version" "$version"
+	  version=`gfortran --version | head -1 | awk '{print $4}'`
+	  assert_equal "$my_mod_version" "$version"
     elif [[ "$my_family" =~ "llvm"  ]];then
-	version=`clang --version | head -1 | awk '{print $3}'`
-	assert_equal "$my_mod_version" "$version"
-	version=`clang++ --version | head -1 | awk '{print $3}'`
-	assert_equal "$my_mod_version" "$version"
-	version=`flang --version | head -1 | awk '{print $3}'`
-	assert_equal "$my_mod_version" "$version"
+	  version=`clang --version | head -1 | awk '{print $3}'`
+	  assert_equal "$my_mod_version" "$version"
+	  version=`clang++ --version | head -1 | awk '{print $3}'`
+	  assert_equal "$my_mod_version" "$version"
+	  version=`flang --version | head -1 | awk '{print $3}'`
+	  assert_equal "$my_mod_version" "$version"
     else
-	flunk "Unsupported compiler toolchain"
+	  flunk "Unsupported compiler toolchain"
```

Comment:
Please undo the re-indentation. The file is a mess concerning indentation and partly adapting one standard does not seem useful in the context of this PR. This probably needs to be changed  for the whole `tests` directory at some point.